src/components/Filters/Filters.tsx

```diff
@@ -55,6 +55,10 @@ export interface FilterInterface {
   shortcut?: boolean;
   /** Whether or not the filter is disabled */
   disabled?: boolean;
+  /**
+   * @default true
```

Comment:
Why should this default to true?  It's a breaking change if so and might come be unexpected behaviour for consumers?